<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>925</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>D2Coding</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>D2Coding</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="font">
     <font>
      <family>D2Coding</family>
     </font>
    </property>
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionLinear_Analysis"/>
    <addaction name="actionMultiple_Linear_Analysis"/>
    <addaction name="separator"/>
    <addaction name="actionSoftmax_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuResult">
    <property name="title">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuResult"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionLinear_Analysis">
   <property name="text">
    <string>Linear Analysis</string>
   </property>
  </action>
  <action name="actionMultiple_Linear_Analysis">
   <property name="text">
    <string>Multiple Linear Analysis</string>
   </property>
  </action>
  <action name="actionSoftmax_Analysis">
   <property name="text">
    <string>Softmax Analysis</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
